ssh/tailssh/tailssh.go

```diff
@@ -1651,20 +1656,29 @@ type loggingWriter struct {
 	r   *recording
 	dir string    // "i" or "o" (input or output)
 	w   io.Writer // underlying Writer, after writing to r.out
+
+	// recordingFailed is whether we should skip recording.
+	// It is set to true if we fail to write to r.out and r.failOpen is set.
+	recordingFailed bool
```

Comment:
Minor thing: Maybe `recordingFailedOpen` just to reinforce that this means not only the failure but also that we checked openness?

Reply:
good point. done